src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingTemplate.cs

```diff
@@ -99,28 +129,26 @@ public string Bind(IReadOnlyDictionary<string, string> parameters)
                 parameters = caseInsensitive;
             }
 
+            StringBuilder builder = new StringBuilder();
             foreach (BindingTemplateToken token in Tokens)
             {
                 if (token.IsParameter)
                 {
-                    string value;
-                    BindingParameterResolver resolver = null;
```

Comment:
All this internal binding logic has been factored out into a DefaultParameterResolver.